PurchasesCoreSwift/Logging/Strings/Emojis.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Emojis.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/8/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum Emojis: String {
+    case appleError = "\u{1F34E}\u{FE0F}" // apple emoji
```

Comment:
these aren't really emojis, we're attaching meaning to them. 
I think the _meaning_ or intent should be a responsibility of `LogIntent`, since that's what it does, and this enum should just be a mapping of emoji to unicode, i.e.: 

```
enum Emoji: String { 
    case apple = "\u{1F34E}\u{FE0F}"
    case heartCatEyes = "\u{1F63B}\u{FE0F}"
    ...
```

Note that this also removes the need for comments on each line, since it's perfectly clear what each one does

Reply:
and then `LogIntent` would do something like: 

```swift
switch self {
         case .appleError: return Emoji.apple.rawValue + Emoji.doubleExclamation.rawValue
```

Which makes total sense, it's essentially saying `appleError = apple + !!`

Reply:
This makes sense, I'll also make updates to the Android version to keep implementation consistent